libs/cdk/virtual-table/src/table-builder.module.ts

```diff
@@ -51,56 +51,56 @@ import { TableBuilderComponent } from './table-builder.component';
     ],
     declarations: [
```

Comment:
see `@angular-eslint/sort-ngmodule-metadata-arrays`, maybe we can enable that

Reply:
there were rare cases when sorting broke the build, let me take it as a separate task